src/Stratis.Bitcoin.Features.SignalR/Startup.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Stratis.Bitcoin.Features.SignalR
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors();
+            services.AddSignalR();
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            app.UseCors(builder => builder
```

Comment:
We should probably consider tightening this down a bit instead of allowing all request, headers and origins